### PR TITLE
Lit element

### PR DESCRIPTION
**Pull request by dlockhart**
I won't merge this yet as I'm still tracking one remaining place in a FRA that needs to be fixed, but it's ready for review with the hope of merging tomorrow!

### CODE REVIEW

**Comment by dlockhart** [comment]
All the places that were relying on `iron-iconset-svg` or setting custom properties from non-components have been fixed up. This is now ready to merge.

File: tier1-icons.js [hunk 1]
```diff
@@ -1,803 +0,0 @@
-import '@polymer/polymer/polymer-legacy.js';
```
Comment (svanherk) [inline tier1-icons.js]: Won't just deleting these files cause build errors for people that were loading them? Like https://github.com/BrightspaceUI/navigation/blob/74425b5736614d97a39e91cfc3322ab3ae4b5df0/d2l-navigation-button-close.js#L4

Comment (svanherk) [reply tier1-icons.js]: Nvm, you're just emptying these files but they aren't being deleted.

Comment (svanherk) [reply tier1-icons.js]: The github diff doesn't make that super clear, everything looks the same other than the small message under "Load Diff":
image
🤷‍♀

Comment (dlockhart) [reply tier1-icons.js]: Haha weird -- yeah just emptying them out.

**Pull request merged**